### ISSUE TITLE
Nested owned types on derived types are mapped to a different table.

### ISSUE THREAD

**Issue by AndriySvyryd** [dotnet/efcore#19279]
Uncomment `ToTable` calls as a workaround.

```C#
public class ApplicationDbContext : IdentityDbContext
{
    public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
        : base(options)
    { }

    protected override void OnModelCreating(ModelBuilder builder)
    {
        base.OnModelCreating(builder);

        builder.Entity<ApplicationUser>(e => {
            e.OwnsOne(au => au.Name, x => {
                //x.ToTable("AspNetUsers");
            });
        });

        builder.Entity<Patient>(e => {
            e.OwnsOne(p => p.Address, x => {
                //x.ToTable("AspNetUsers");
            });
            e.OwnsOne(p => p.EmergencyContact, x => {
                //x.ToTable("AspNetUsers");
            });

            e.OwnsOne(p => p.ResponsibleParty, x => {
                //x.ToTable("AspNetUsers");
                x.OwnsOne(rp => rp.Address, rpa => {
                    //rpa.ToTable("AspNetUsers");
                });
            });
        });
    }

    public DbSet<Patient> Patients { get; set; }
}
```

```C#
public class ApplicationUser : IdentityUser
{
    public PersonalName Name { get; set; }
}

public class Patient : ApplicationUser
{
    public PostalAddress Address { get; set; }
    public EmergencyContact EmergencyContact { get; set; }
    public ResponsibleParty ResponsibleParty { get; set; }
}

public class PostalAddress
{
    public string Address { get; set; }
}

public class EmergencyContact
{
    public string Name { get; set; }
}

public class ResponsibleParty
{
    public string Name { get; set; }
    public PostalAddress Address { get; set; }
}
```

[DotNet3.0.zip](https://github.com/aspnet/EntityFrameworkCore/files/3951938/DotNet3.0.zip)

**Comment by AndriySvyryd** [comment]
Might be related to https://github.com/dotnet/efcore/issues/19543

**Comment by smitpatel** [comment]
Observations:
- The original repro is MVC app with identity and does not work out of the box. It has several obsolete warnings and throws and error with UseMvc method. The original repro also targets .NET 3.0. I am not able to get repro of error in it.
- I created new MVC app with identity targeting .NET 3.1
  - Added owned classes as detailed above and configured them without ToTable calls.
  - Generated new migration.
  - Result is all columns are added in AspNetUsers table only.

@AndriySvyryd - Do you have anything specific in mind which was failing? This seems to be working 3.1

**Comment by AndriySvyryd** [comment]
@smitpatel Even if it was failing in some 3.x version, this category of issues is fixed in 5.0 by fb33fe4
